src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+
+    private static final FigureSupplier figureSupplies = new FigureSupplier();
+
+    public static void main(String[] args) {
+
+        Figure getRandomFIgure = figureSupplies.getRandomFigure();
+        Figure getDefaultFigure = figureSupplies.getDefaultFigure();
```

Comment:
Similarly, the variable name `getDefaultFigure` does not follow Java naming conventions. Consider renaming it to `defaultFigure`.